rules/group.go

```diff
@@ -865,6 +870,14 @@ func NewGroupMetrics(reg prometheus.Registerer) *Metrics {
 			},
 			[]string{"rule_group"},
 		),
+		GroupLastRestoreDuration: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "rule_group_last_restore_duration_seconds",
+				Help:      "The duration of the last rule group restoration.",
```

Comment:
do you want to add some details what restoring refers to? Otherwise these metrics are pretty opaque to anyone who hasn't read the code.

Reply:
Fixed in https://github.com/prometheus/prometheus/pull/13974/commits/5beb2fe0051fb0ea04e32ab0e0b8bdac86d3ae75